libs/utils/src/lib.rs

```diff
@@ -67,7 +67,7 @@ macro_rules! failpoint_sleep_millis_async {
         let should_sleep: Option<std::time::Duration> = (|| {
             fail::fail_point!($name, |v: Option<_>| {
                 let millis = v.unwrap().parse::<u64>().unwrap();
-                Some(Duration::from_millis(millis))
+                Some(std::time::Duration::from_millis(millis))
```

Comment:
NIT: I think leading `::` is the way to go here.
Same for all other types in this macro.

Or, `use ::std as std;` at the beginning of the block.

https://stackoverflow.com/a/65955962

@koivunej would probably know more.

Reply:
This is being discussed at https://github.com/neondatabase/neon/pull/4195. I just needed something here to make the CI pass, but will commit PR #4195 before this.